lib/recording/RecordingManager.js

```diff
@@ -54,7 +54,10 @@ class RecordingManager {
     getRecordingEntrySession(entryObj, mp4FilesQueue = null) {
         let entryId = entryObj.entryId;
         if (mp4FilesQueue != null && mp4FilesQueue.length > 0) {
-            logger.debug(`[${entryObj.entryId}-${entryObj.recordedEntryId}] getRecordingEntrySession called with mp4 files queue size [${mp4FilesQueue.toString()}]`);
+            let chunksNameChained = _.reduce(mp4FilesQueue, function (str, obj) {
+                return str + obj.chunkName + " ";
+            }, "");
+            logger.debug(`[${entryObj.entryId}-${entryObj.recordedEntryId}] getRecordingEntrySession called with mp4 files queue size [${mp4FilesQueue.length}][${chunksNameChained}]`);
```

Comment:
please remove, it can create lots of log lines, we don't really care about the chunk names...